logic/src/commonMain/kotlin/com/wire/kalium/logic/sync/incremental/EventGatherer.kt

```diff
@@ -107,27 +115,27 @@ internal class EventGathererImpl(
         }
 
     private suspend fun FlowCollector<Event>.onWebSocketEventReceived(webSocketEvent: WebSocketEvent.BinaryPayloadReceived<Event>) {
-        kaliumLogger.withFeatureId(SYNC).i("SYNC: Websocket Received binary payload")
+        logger.i("SYNC: Websocket Received binary payload")
```

Comment:
Isn't it redundant prefix with SYNC now you have a logger with featureId("SYNC")

Reply:
Indeed it is now.
I'll take these out.